frontend/src/app/middleware/middleware.service.ts

```diff
@@ -51,24 +51,24 @@ export class MiddlewareService {
   getClaim(vin: string): Observable<Protos.Claim> {
 
     // TODO: Remove mocked data.
-    this.x = {
-      vehicle: {
-        vin: vin,
-        model: "Escape",
-        color: "Red",
-        year: 2005,
-        miles: 100023
-      },
-      status: 0,
-      files: [
-        {
-          hash: "1237-1232-1231fsef-123-sgseg",
-          name: "police-report.png"
-        }
-      ]
-    }
-
-    return of<Protos.Claim> (this.x);
+    // this.x = {
```

Comment:
Commenting out mocks so it can be run against real data - confirmed by @sloan-seamus to be connecting and getting data back successfully (can verify by navigating to `/claims/{added-vin-number}`. i will add a view claim button on the view page to make it easier to get to this page tonight